package.json

```diff
@@ -39,6 +39,11 @@
     "packages/*"
   ],
   "dependencies": {
+    "@fullcalendar/core": "^6.1.9",
+    "@fullcalendar/daygrid": "^6.1.9",
+    "@fullcalendar/interaction": "^6.1.9",
+    "@fullcalendar/react": "^6.1.9",
+    "@fullcalendar/timegrid": "^6.1.9",
```

Comment:
We already have a calendar?

Reply:
Not one with all the features we want (I think). It should show both the date and the time of day at once, and support highlighting/marking parts of the calender

Reply:
I am very much against having multiple calendars in the webapp, especially one that requires 5 packages.

Reply:
But the calender we already use doesn't have what we need at all

Reply:
If we were to use it, you would not be able to display when an object is booked in a calender view (or just display it by the date, but not the time of day). Also, either we would have to say that just one person can lend an object each day, or, because there's no smooth way of displaying what times the object is busy, the user would have to type in the time of day manually and then just guess when the object is avalible. Sure, you could find some half-assed solutions to make it a bit more user friendly than that, but stil. I think it's better to import a few more packages, and get a better user experience. I imagine that if the ledning system isn't smooth enought, Arrkom and PR who already have systems for lending, will just keep on using their own system instead of using ours.